src/RestClient/endpoints/V3/Themes/Themes.ts

```diff
@@ -0,0 +1,58 @@
+import { AxiosInstance } from 'axios';
+
+import { AxiosPromise } from '../../../../types';
+
+import type { Theme } from './types';
+
+export const themesPath = 'v3/themes';
+
+class Themes {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Gets a list of store Themes
+   *
+   * @returns Promise resolving to a list of categories
+   */
+  list(): Theme['ListResponse'] {
+    return this.client.get(themesPath);
+  }
+
+  /**
+   * Uploads a new theme to your BigCommerce store.
+   *
+   * @param file The theme file
+   * @returns Promise resolving to a newly created store theme
```

Comment:
Again, this is not accurate - the promise will resolve to a JSON body containing a `job_id`